src/reductiveContext.re

```diff
@@ -8,35 +8,50 @@ module type Config = {
 
 module Make = (Config: Config) => {
   open Subscription;
-  let storeContext = React.createContext(Config.store);
+  let storeContext = React.createContext(None);
```

Comment:
Btw, if we are not going to use the store passed in `Config`, we should probably remove it from there.

Reply:
I'm not sure I understand want you mean. We still need an "invalid" reference to the `storeContext`, so it needs to be initialised, no?

Reply:
This is how `Config` that is passed in `Make` is defined:
```reason
module type Config = {
  type state;
  type action;
  let store: Reductive.Store.t(action, state);
};
```

But if now we aren't going to use the value of `store` from the config, and instead pass it in `Provider`, then we should probably remove it from the config.
Does this make sense?

Reply:
oh yes, good point!

Reply:
Yup - 100% agreed here. Down to remove from config and then require passing in to the component. Didn't notice this when I made my earlier comments.